recipe/meta.yaml

```diff
@@ -142,6 +144,34 @@ outputs:
         - jax  >0.3.0
         - requests
 
+  - name: {{ name|lower }}-mujoco
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python  {{ python_min }}
+      run:
+        - {{ pin_subpackage('adam-robotics', max_pin='x.x.x') }}
+        - mujoco
+        - robot_descriptions
+        - python  >={{ python_min }}
+    test:
+      source_files:
+        - tests
+      commands:
+        - pytest ./tests/test_mujoco.py -v
+      requires:
+        - python  {{ python_min }}
+        - pip
+        - pytest
+        - icub-models
+        - idyntree
+        - xorg-libxxf86vm
+        - git
+        - gitpython
+        - requests
+        - mujoco
+        - robot_descriptions
```

Comment:
I always understood that the `adam-robotics-<something>` matched the `adam-robotics[something]` optional dependency in https://github.com/ami-iit/adam/blob/219991c6c646ce0c6f1aa1251279e7906d7b4753/pyproject.toml#L28-L35 . As there is no `mujoco` optional dependency group, perhaps we can avoid to have the `adam-robotics-mujoco` output?